image/README.md

```diff
@@ -0,0 +1,119 @@
+# @magenta/image
+
+This JavaScript implementation of Magenta's image models uses [TensorFlow.js](https://js.tensorflow.org) for GPU-accelerated inference.
+
+Complete documentation is available at https://tensorflow.github.io/magenta-js/image.
+
+## Contents
+
+- [Example Applications](#example-applications)
+- [Supported Models](#supported-models)
+- [Getting Started](#getting-started)
+
+## Example Applications
+
+Here are a few applications built with `@magenta/image`:
+
+- Coming soon!
+
+You can also try our [hosted demos](https://tensorflow.github.io/magenta-js/image/demos) for each model and have a look at the [demo code](./demos).
+
+## Supported Models
+
+### Fast Arbitrary Image Stylization
+
+Implements Ghiasi et al.'s fast arbitrary style transfer model ([paper](https://arxiv.org/abs/1705.06830), [code](https://github.com/tensorflow/magenta/tree/master/magenta/models/arbitrary_image_stylization)). Wraps around Reiichiro Nakano's [TensorFlow.js port](https://github.com/reiinakano/arbitrary-image-stylization-tfjs) of the model checkpoint.
+
+**Demo Application:** Coming soon!
+
+## Getting started
+
+There are two main ways to get MagentaImage.js in your JavaScript project:
+via [script tags](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_JavaScript_within_a_webpage) **or** by installing it from [NPM](https://www.npmjs.com/)
+and using a build tool like [yarn](https://yarnpkg.com/en/).
+
+### via Script Tag
+
+Add the following code to an HTML file, and place a content (`content.jpg`) and style (`style.jpg`) image in the same directory:
+
+```html
+<html>
+  <head>
+    <!-- Load @magenta/image -->
+    <script src="https://cdn.jsdelivr.net/npm/@magenta/image@^1.0.0"></script>
+  </head>
+  <body>
+    <img id="content" src="content.jpg"/>
+    <img id="style" src="style.jpg"/>
+    <canvas id="stylized"></canvas>
+      <script>
+        const model = new mi.ArbitraryStyleTransferNetwork();
+        const contentImg = document.getElementById('content');
+        const styleImg = document.getElementById('style');
+        const stylizedCanvas = document.getElementById('stylized');
+
+        async function stylize() {
+          const bottleneck = await mi.tf.tidy(() => {
```

Comment:
why is this called bottleneck? is there a more descriptive word, maybe? like `styleParameters`? (i know nothing about this model)

Reply:
That's the language used in the paper (this corresponds to the output of a "bottleneck" layer just before the linear projection onto the instance normalization parameter values), but I don't feel very strongly about keeping that name and agree that a more descriptive name would be better.

I changed the name to `styleRepresentation`.